src/components/Snackbar.tsx

```diff
@@ -207,6 +212,7 @@ const Snackbar = ({
             styles.content,
             { marginRight: action ? 0 : 16, color: colors.surface },
           ]}
+          numberOfLines={numberOfLines}
```

Comment:
According to the MaterialDesign guidelines we shouldn't allow to customize the number of lines. Documentation indicates that on mobile number of lines should be up to two and on the web it should be single line. Then basically value should be set conditionally:

`numberOfLines={Platform.OS === 'web' ? 1 : 2}`

Reply:
What do you think about setting the value conditionally only if no value was provided? I believe this would achieve the best of both worlds.

```suggestion
          numberOfLines={
            numberOfLines > 0
              ? numberOfLines
              : Platform.OS === 'web' ? 1 : 2
          }
```